dae/dae/configuration/dae_config_parser.py

```diff
@@ -124,4 +124,15 @@ def parse(cls, config, dae_data_dir=None):
                 if config.genomic_scores_db else None,
         }
 
+        if config.gpfjs:
+            filepath = config.gpfjs.permission_denied_prompt
+            assert os.path.exists(filepath), filepath
+            with open(filepath, 'r') as prompt_file:
+                config.gpfjs.permission_denied_prompt = prompt_file.read()
+        else:
```

Comment:
You should probably consider moving this to the DEFAULT_SECTION_VALUES.

Reply:
:+1:

Reply:
on second thought, I don't like using the same attribute for both the filepath and the contents of the file; I'll fix it and re-request the review